airflow/kubernetes/pod_generator_deprecated.py

```diff
@@ -187,15 +187,15 @@ def __init__(
         self.spec = k8s.V1PodSpec(containers=[])
         self.spec.security_context = security_context
         self.spec.tolerations = tolerations
-        self.spec.dns_policy = dnspolicy
+        self.spec.dns_policy = dnspolicy if dnspolicy else "Default"
```

Comment:
The default is actually `ClusterFirst` (see https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#hostname-and-name-resolution).

Reply:
(Yeah, confusing that "default" isn't the default 🤷‍♂️)

Reply:
Well.. That's ..... unexpected :D. 

So you say "Default" is not "default" :facepalm: